src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -276,8 +276,10 @@ public void onTick(TickEvent.ServerTickEvent event) {
     @SideOnly(Side.CLIENT)
     @SubscribeEvent
     public void onTick(TickEvent.ClientTickEvent event) {
-        if (event.phase == TickEvent.Phase.START) {
-            this.scheduler.tickSyncScheduler();
+        if (!net.minecraft.client.Minecraft.getMinecraft().isIntegratedServerRunning()) {
```

Comment:
Fully qualified import is un-needed as import lines are just for our benefit. When the code is compiled, everything is made fully qualified.

Reply:
TIL, ty.